Library/Homebrew/tab.rb

```diff
@@ -250,6 +250,11 @@ def to_s
     when true  then s << "Poured from bottle"
     when false then s << "Built from source"
     end
+    unless time.nil?
+      s << "Installed" if s.empty?
```

Comment:
> I think it would be best to make this an else clause in the above case statement.

Well actually no, because "Installed" shouldn't be printed if `time` is not available and `used_options` is empty. I'm not sure if a missing `time` is possible though. If that's impossible, then we can certainly move `s << "Installed" if s.empty?` into an else clause.